### PR TITLE
Adjustments needed in make_rootfs for using it in Firecracker

### PR DESCRIPTION
**Pull request by dianpopa**

### CODE REVIEW

File: resources/disk/make_rootfs.sh [hunk 1]
```diff
@@ -37,12 +39,14 @@ Options:
   -d debdir         Directory containing .deb packages for the Linux image.
   -w workdir        Working directory for the kernel build.
   -o diskfile       Name of the resulting disk file.
-  -f flavour        The Ubuntu flavour desired. Defaults to "focal"
+  -f flavour        The Ubuntu flavour desired. Defaults to "focal".
+  -f disk size      The size of the resulting rootfs.
```
Comment (andreeaflorescu) [inline resources/disk/make_rootfs.sh:43]: We should also add a comment about what is the size unit (or just give a few examples so people know how to use it).

File: resources/disk/make_rootfs.sh [hunk 2]
```diff
@@ -22,13 +22,15 @@ DISKFILE="rootfs.ext4"
 # Directory containing .deb packages for the Linux image.
 DEBDIR=
 # Disk size. Currently hardcoded to 1 GiB.
-DISKSIZE="1G"
+DISKSIZE="1000M"
```
[the remark this answers is not in the document]
Comment (dianpopa) [reply resources/disk/make_rootfs.sh]: This change is not needed.

File: resources/disk/make_rootfs.sh (hunk 1 above)
Comment (dianpopa) [reply resources/disk/make_rootfs.sh]: Done.

**Comment by dianpopa** [comment]
@andreeaflorescu @lauralt can you please take a look at this one, too?

**Review by andreeaflorescu — APPROVED** [review]
Sorry about the delay, I somehow missed the notification. LGTM!

File: resources/disk/make_rootfs.sh [hunk 3]
```diff
@@ -29,19 +29,25 @@ DISKMNT="mnt/rootfs"
...
-Usage: $(basename $SOURCE) -d debdir [-w workdir] [-o diskfile] [-c]
+Usage: $(basename $SOURCE) -d debdir [-w workdir] [-o diskfile] [-v version] [-s size] [-c] [-n]
 
 Options:
   -d debdir         Directory containing .deb packages for the Linux image.
   -w workdir        Working directory for the kernel build.
   -o diskfile       Name of the resulting disk file.
+  -v version        The Ubuntu version desired. Defaults to focal.
+  -s disk size      The size of the resulting rootfs. This needs to be
+                    an integer followed by an unit (10K, 500M, 1G).
```
Comment (lauralt) [inline resources/disk/make_rootfs.sh:44]: nit: I think the unit is not mandatory.

**Review by lauralt — APPROVED** [review]

**Pull request merged**